src/BuiltInTools/dotnet-watch/HotReload/BlazorWebAssemblyDeltaApplier.cs

```diff
@@ -59,12 +66,24 @@ async Task<ImmutableArray<string>> GetApplyUpdateCapabilitiesCoreAsync()
                     }
 
                     var capabilities = Encoding.UTF8.GetString(buffer.AsSpan(0, response.Value.Count));
+                    var shouldFallBackToDefaultCapabilities = false;
 
                     // error while fetching capabilities from WASM:
                     if (capabilities.StartsWith("!"))
                     {
-                        _reporter.Error($"Exception while reading WASM runtime capabilities: {capabilities[1..]}");
-                        return ImmutableArray<string>.Empty;
+                        _reporter.Warn($"Exception while reading WASM runtime capabilities: {capabilities[1..]}");
```

Comment:
I downgraded this to a warning since it no longer indicates a fatal error.

Reply:
@tmat Should we remove the `console.warn(error)` as well? I'm thinking that might be good since it will be a common warning and customers might get annoyed that they can't get rid of it.